### PR TITLE
qotd

### PR DESCRIPTION
**Pull request by callumjt**
adds question of the day

hoping the code works :)

### CODE REVIEW

File: modules/reminders/send.ts [hunk 1]
```diff
@@ -232,6 +233,9 @@ async function sendReminders(): Promise<NodeJS.Timeout | undefined> {
 					});
 					continue;
 				}
+				case SpecialReminders.QOTD: {
+					await sendQOTD();
+				}
```
Comment (RedGuy12) [inline modules/reminders/send.ts:238]:
```suggestion
continue; }
``` oops

**Comment by callumjt** [comment]
alr now please tell me you hate my code :3

**Comment by RedGuy12** [comment]
i meant custom emojis per question 😭 like how /poll works kinda

**Comment by callumjt** [comment]
> i meant custom emojis per question 😭 like how /poll works kinda

oh...

**Comment by callumjt** [comment]
So should there always be emojis?

**Pull request merged**